static/sass/print.scss

```diff
@@ -91,7 +95,12 @@ footer,
 .global-nav__header-list,
 .dropdown-window,
 .dropdown-window-overlay,
-.p-navigation--secondary {
+.p-navigation--secondary,
+.p-sticky-toc {
+  display: none;
+}
+
+#canonical-global-nav {
```

Comment:
I asked you to add this but the class `.global-nav` in the selector above already applies this rule so this is not needed.